### ISSUE TITLE
GGMAX Bug - DLC Assett - Material Rendering Bug

### ISSUE THREAD

**Issue by Kitakazi** [TheGameCreators/GameGuruRepo#5717]
Hi, I loaded up a level and noticed the material on a wall was not correct. This is a wall from the Offworld DLC, not the stock one that comes with it but a re-imported version. I reimported it so that I could retexture it. This was a while ago probably the time the Offworld pack came out. When I re-imported this wall, I gave it new textures and I placed it in my own folder completely different than the Offworld DLC. 

I have had no issues untill recently when loaded up a level and noticed the material showing on the wall was not correct. This wall should not  have "Enable custom materials" turned on. Which it does not. But when I select the wall it is showing completely different material settings than what is in the FPE. The textures are correct but the settings such a normal strength, roughness strength are not correct. 

Here is where it gets really weird. Once I click the wall and then click off of it the material settings completely change. But neither material settings showing are correct. If I enable custom materials it does not show the correct values for normal strength and roughness/metalness setting. 

I tried reimporting this entity and renaming the textures but it still has the exact same problem. Here is a video showing this. 
https://youtu.be/Jn7UfD7p4W0?si=tLY570EOgD5XhjxV

So in short the entity is placed into the level with "enable custom materials" turned on by default, even though it should not be. Turning off enable custom materials will cause the entity to display a material that is NOT what the FPE settings are. At this point the entity will then change to a completely different material setting when the entity has been deselected. 

This is also a big problem for me because I use this wall ALOT on a bunch of different levels. Over all the levels I currently have this wall is probably used well over a thousand times.

**Comment by davetgc** [comment]
Can you please email the custom entity to Lee@thegamecreators.com, and point to this issue.  Also, can you say which version of MAX you're using (Release, EXP, or DEV build) and post your system specs.  Thanks.

**Comment by Kitakazi** [comment]
I just sent the model over. I'm on the EXP build. 

Specs: 
13th Gen Intel(R) Core(TM) i7-13700KF   3.40 GHz
32.0 GB DDR5
NVIDIA GeForce RTX4080

**Comment by LeeBamberTGC** [comment]
@Kasseyus Can you obtain this custom wall and reproduce with a fresh blank level, I can then run through your steps in quick order and resolve this material issue, thanks!

**Comment by Kasseyus** [comment]
@Kitakazi Can you please forward the affected asset to me at chris@thegamecreators.com, and I will be able to take a closer look.

**Comment by Kitakazi** [comment]
Sent.

**Comment by Kitakazi** [comment]
@Kasseyus Were you able to reproduce this issue?

**Comment by Kasseyus** [comment]
@Kitakazi Hi.  I'm currently investigating this issue, and while I've made some progress I have not yet re-produced the exact bug you are encountering.

From what I've been able to determine, this particular model has its panels split into 4 separate meshes, although all using the same texture, and the specific panel that is being affected is the first mesh in the list.

Would you be able to create a new video showing this issue, but only using this asset in a fresh scene?  This will remove the distractions of the other entities in the scene and help narrow down exactly what causes this bug to occur.

Also, if you could E-Mail me the FPM of the affected level to chris@thegamecreators.com I can have a look first-hand.

**Comment by Kitakazi** [comment]
There was an update a day  or so ago and ever since then this issue seems to have been fixed. So yea I guess this one is good.
